1.9/installing/custom/system-requirements/index.md

```diff
@@ -131,7 +131,25 @@ Add the following line to your `/etc/sudoers` file. This disables the sudo passw
 
 Alternatively, you can SSH as the root user.
 
-### Enable NTP
+### Enable Time synchronization
+
+Time synchronization is a core requirement of DC/OS. There are various methods
+of ensuring time sync. NTP is the typical approach on bare-metal.
+Many cloud providers use hypervisors which push time
+down to the VM guest operating systems. In certain circumstances hypervisor
+time-sync may conflict with NTP.
+
+You must understand how to properly configure time synchronization for your
+environment. When in doubt, enable NTP and check using `/opt/mesosphere/bin/check-time`.
+
+#### Enable Check Time
+
+There is an `ENABLE_CHECK_TIME` environment variable that needs to be set
```

Comment:
You must set the `ENABLE_CHECK_TIME` environment variable in order for `/opt/mesosphere/bin/check-time` to function. We recommend you enable this globally....